<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-08-05T07:01:24Z" name="Default suite" finished-at="2019-08-05T08:42:11Z" duration-ms="6047136">
    <groups>
    </groups>
    <test started-at="2019-08-05T07:01:24Z" name="Default test" finished-at="2019-08-05T08:42:11Z" duration-ms="6047136">
      <class name="TestCases.PatientEMRTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:TestCases.PatientEMRTest@358c99f5]" started-at="2019-08-05T07:01:24Z" name="setUp" finished-at="2019-08-05T07:01:33Z" duration-ms="9088" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="test(java.lang.String)[pri:1, instance:TestCases.PatientEMRTest@358c99f5]" started-at="2019-08-05T07:01:33Z" name="test" data-provider="MrNoProvider" finished-at="2019-08-05T07:13:06Z" duration-ms="692537" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[UA1300000063641]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method signature="test(java.lang.String)[pri:1, instance:TestCases.PatientEMRTest@358c99f5]" started-at="2019-08-05T07:13:06Z" name="test" data-provider="MrNoProvider" finished-at="2019-08-05T07:13:37Z" duration-ms="31204" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[UA1300000048504]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method signature="test(java.lang.String)[pri:1, instance:TestCases.PatientEMRTest@358c99f5]" started-at="2019-08-05T07:13:37Z" name="test" data-provider="MrNoProvider" finished-at="2019-08-05T08:42:11Z" duration-ms="5314235" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[UA1300000026068]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The links in which exception occored are :- "Speech Therapy Screening - (07-08-2016), " expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The links in which exception occored are :- "Speech Therapy Screening - (07-08-2016), " expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at TestCases.PatientEMRTest.test(PatientEMRTest.java:54)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- TestCases.PatientEMRTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
